src/app_service/service/wallet_account/dto.nim

```diff
@@ -147,6 +147,13 @@ proc getAddress*(self: WalletTokenDto): string =
 
   return ""
 
+proc getTotalBalanceOfSupportedChains*(self: WalletTokenDto): float64 =
+  var sum = 0.0
+  for chainId, balanceDto in self.balancesPerChain:
+    sum += balanceDto.balance
```

Comment:
Should we use some some of bigInt for that? I'm worried it might overflow with the crazy decimals.

Reply:
Had a big discussion around that with @alaibe 
I think the biggest supported float is big enough to not worry to much about that here.
OTOH I don't think there's a theoretical decimals limit, so there could indeed be a token that goes bananas here.

It would require us to switch to `u256` all the way down tho. Which would also require a big refactor of the wallet if I'm not mistaken.